assets/app/views/settings.html

```diff
@@ -34,10 +34,10 @@
               Status has not been reported on this quota usage record.  Any resources limited by this quota record can not be allocated.
             </td>
           </tr>
-          <tr ng-class="{warning: quota.status.used && quota.status.used[resourceType] >= quota.status.hard[resourceType]}" ng-repeat="(resourceType, specMax) in quota.spec.hard">
+          <tr ng-class="{warning: (quota.status.used[resourceType] | usageValue) >= (quota.status.hard[resourceType] | usageValue)}" ng-repeat="(resourceType, specMax) in quota.spec.hard">
```

Comment:
nope because quota.status.hard is not necessarily the same as quota.spec.hard,  spec is what you wanted if you are allowed, status is what you are actually allowed